plugins/string/mod.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * Derived from <https://github.com/TrySound/rollup-plugin-string>
+ * 
+ * Licensed as follows:
+ * 
+ * The MIT License (MIT)
+ * 
+ * Copyright (c) Bogdan Chadkin <trysound@yandex.ru>
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import type { FilterPattern, Plugin } from "../../mod.ts";
+
+import { createFilter } from "../../src/rollup/createFilter.ts";
+
+export interface RollupStringOptions {
+
+  /**
+   * Specify include files
+   */
+  include?: FilterPattern;
```

Comment:
If not providing an `include` value results in a thrown error, perhaps it would be best the type is not optional

```suggestion
  include: FilterPattern;
```